packages/components/card/src/card.tsx

```diff
@@ -55,7 +55,15 @@ const Card = (props: TCardProps) => (
       // This is mostly useful in case custom styles are targeting this element.
       <div>{props.children}</div>
     ) : (
-      <Inset scale={props.insetScale}>{props.children}</Inset>
+      <div
+        css={css`
+          > div {
```

Comment:
I was thinking that maybe this rule should be applied to the `Inset` component instead.
What do you think?

Reply:
Thinking about it now, I think that should work fine too, I was not so sure of how this would be of effect to other use cases of the `Inset` component. That's why I tried to keep it within the scope of the cards use case. 
updated here: https://github.com/commercetools/ui-kit/pull/2624/commits/b54dd8e22fd387d7a9ddb8885b97386dbf9023ed

Thank you

Reply:
Let's see what our teammates think about it.

Reply:
I'm not aware of any downsides. Perhaps we should consider updating `<InsetSquish>` as well for consistency 🤷‍♂️

Reply:
Good idea @kark, looking at it, I think we should add it there too. 👍🏾

Reply:
updated here since I also have checked and don't see any reason why we should not update` Inset squish`  also: https://github.com/commercetools/ui-kit/pull/2624/commits/8af1368ae3411f08b3b5e7c92f042856e19c4c45